### ISSUE TITLE
Keeweb not jumping to the foreground on password fields in macOS Monterey

### ISSUE THREAD

**Issue by dploeger** [keeweb/keeweb#1932]
**Describe the bug**
I recently updated to macOS Monterey and for some reason, Keeweb stays in the background when using Autotype on password fields.

**To Reproduce**

1. (on macOS Monterey)
2. Open a web form that has a password field
3. Select that password field
4. Hit autotype
5. Keeweb stays in the background

**Expected behavior**
Keeweb jumps to the foreground (like on any non password field) and shows the autotype selector

**Environment**
KeeWeb v1.18.7 (dfc6201, 2021-07-18)
Environment: electron v12.0.7
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 12_0_1) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.18.7 Chrome/89.0.4389.128 Electron/12.0.7 Safari/537.36

**Logs**

```
index.html:1 2021-11-17T06:32:48.874Z [auto-type] Getting window info
index.html:1 2021-11-17T06:32:48.888Z [auto-type] Window info undefined Anmeldung | Splunk - Google Chrome – Dennis https://prd-p-3xh55.splunkcloud.com/de-DE/account/login?session_expired=1&return_to=%2Fde-DE%2Fapp%2Fmonitoringkubernetes%2Fkube_cluster_audit%3Fform.period.earliest%3D-60m%2540m%26form.period.latest%3Dnow%26form.refresh%3D0%26form.kubernetes_cluster_eval%3D*%26form.kubernetes_node_label%3D*%26form.host%3D*%26form.username%3D*
index.html:1 2021-11-17T06:32:48.888Z [auto-type] processing auto-type event
index.html:1 2021-11-17T06:32:54.760Z [auto-type] Entry selected Object
index.html:1 2021-11-17T06:32:54.761Z [auto-type] Hide window
index.html:1 2021-11-17T06:32:54.868Z [auto-type] Skipped active window check because window id is unknown
index.html:1 2021-11-17T06:32:54.868Z [auto-type] Start {S:Cloud-Admin-Password}
index.html:1 2021-11-17T06:32:54.868Z [auto-type] Parsed [op:S]
index.html:1 2021-11-17T06:32:54.868Z [auto-type] Resolved [[*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*]]
index.html:1 2021-11-17T06:32:55.054Z [auto-type] Complete 186ms
```

**Comment by antelle** [comment]
Hi! I also updated recently, it does for me!
Maybe it needs to be re-added to the accessibility settings of macOS? 🤔

**Comment by antelle** [comment]
Just to understand, you hit the "global auto-type" hotkey, right?

**Comment by dploeger** [comment]
Hm, maybe. I'll try that out. Yes, global auto-type-

**Comment by dploeger** [comment]
Nope, doesn't work. I even reinstalled the app, but the problem persists.

**Comment by antelle** [comment]
What was the app where you tried to press the auto-type hotkey?

**Comment by dploeger** [comment]
With Google Chrome, Safari and the FortiClient (which is based on Chrome as well AFAIK)

**Comment by antelle** [comment]
Does it work with a non-browser, for example, a text editor?

**Comment by dploeger** [comment]
Ah, actually yes. I tried it on a password field of a mail account and it works there. So it's only password fields in Browsers apparently.

**Comment by antelle** [comment]
Interesting. Just tried on Chrome, seems to work for me. Does it work on the new tab page?

**Comment by dploeger** [comment]
The new tab page? It only happens on password fields. All other fields work fine.

**Comment by antelle** [comment]
I see! Just tried with a focus in a password field, got the same. I tried on a form with username/password before, same for me too.

**Comment by dploeger** [comment]
Ah, right.

**Comment by auprobst** [comment]
Hello

I'm facing the same issue with Firefox, Chrome, MS Teams app, Discord app.

**Comment by dploeger** [comment]
Hmm... Weird. Starting today, everything's back to normal. I didn't (knowingly) change or update anything. Let's see, if it stays this way.

**Comment by dploeger** [comment]
Ah, no. After rebooting, it's back. :(

**Comment by antelle** [comment]
Yeah, same, I thought maybe a new version of macOS fixed it, but it's also like this for me... We need to find a workaround
